src/ng/directive/attrs.js

```diff
@@ -159,7 +159,7 @@
  *
  * @description
  *
- * This directive sets the `disabled` attribute on the element if the
+ * This directive sets the `disabled` attribute on a form element if the
```

Comment:
I think this can be more confusing than helpful. ("form element" often refers to the `<form>` element.)
How about:

> This directive sets the `disabled` attribute on the element (typically a form control) if the...